### PR TITLE
import checks from `apps-ss`

### LINKED ISSUE #37: Import checks from `applications-superstaq`

### PR DESCRIPTION
**Pull request by perlinm**
fixes #37

### CODE REVIEW

File: dev-requirements.txt [hunk 1]
```diff
@@ -1,7 +1,9 @@
-black~=22.3.0
+black[jupyter]~=22.3.0
+flake8-import-order==0.18.1
```
Comment (vtomole) [inline dev-requirements.txt:2]:
```suggestion
flake8-import-order~=0.18.1
```

File: dev-requirements.txt [hunk 2]
```diff
@@ -1,7 +1,9 @@
...
 flake8~=3.8.4
-flake8-import-order~=0.18.1
-mypy==0.910
-pylint~=2.6.0
+mypy~=0.961
+nbmake~=1.3.0
+pylint>=2.13.0
+pytest-cov==2.11.1
```
Comment (vtomole) [inline dev-requirements.txt:7]:
```suggestion
pytest-cov~=2.11.1
```

**Review by vtomole — COMMENTED** [review]
Couple of nits then LGTM

**Comment by perlinm** [comment]
FYI these are all going to become nit after [`app-ss#77`](https://github.com/SupertechLabs/applications-superstaq/pull/77)

File: dev-requirements.txt (hunk 2 above)
Comment (perlinm) [reply dev-requirements.txt]: done

File: dev-requirements.txt (hunk 1 above)
Comment (perlinm) [reply dev-requirements.txt]: done

**Review by vtomole — APPROVED** [review]
LGTM

**Pull request merged**